torchmultimodal/modules/layers/quantization.py

```diff
@@ -27,6 +28,11 @@ class Quantization(nn.Module):
     The embedding weights are trained with exponential moving average updates as described
     in original paper.
 
+    Code was largely inspired by a PyTorch implementation of the author's original code, found here:
```

Comment:
Relocate this paragraph to be just under the license header? Doesn't seem to be specific to `QuantizationOutput`.